### DISCUSSION TITLE
Database Statistics in graph form

### DISCUSSION THREAD

**Discussion by j450n1** [IgnisDa/ryot#696] [Ideas]
Hi there, Firstly this project is fantastic, I recently moved from Media tracker and I am loving it so far. 

Just wondering if it would be possible to take the data that is already collected and display it in graph form? Such as hours watched per day/ week/month or Items added to watch list per day/week/month? Similar to Tautulli.

**Comment by IgnisDa** [comment]
This is already being tracked by #445. I'd appreciate it if you could attach some screenshots there.
